packages/remix-dev/cli/commands.ts

```diff
@@ -107,6 +107,7 @@ export async function watch(
       start = Date.now();
       onRebuildStart && onRebuildStart();
       log("Rebuilding...");
+      broadcast({ type: "REBUILDING" });
```

Comment:
Here you could maybe make this conditional based on whether it's enabled in `remix.config.js`? It could be enabled by default.

Reply:
this would be nice, was thinking to keep broadcasting the event I believe it will be useful as could be used in a different context to show rebuilding and instead to control the behavior in the LiveReload component, what do you think?